examples/vitess/etcd-controller-template.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: etcd-{{cell}}
+  labels:
+    name: etcd
+    cell: {{cell}}
+    app: vitess
+spec:
+  replicas: {{replicas}}
+  selector:
```

Comment:
You want this selector to match pods that are not labeled `app: vitess`?

If you want the selector to match the pod template labels exactly, just omit it.